pkg/config/split.go

```diff
@@ -84,7 +84,28 @@ func ConfigureSetSimple(rest string, cfgname string, field reflect.Value) error
 			v := rest == "true"
 			return reflect.ValueOf(&v), nil
 		case reflect.String:
+			unquoted, err := strconv.Unquote(rest)
```

Comment:
How do you want to handle cases like `"forgot the last quote` ?

My PR handles it as an error (clearly they meant to specify a quoted string).  This PR will treat it as a literal (which is more back-compatible, but less correct.

You decide - the rest LGTM :)